assets/language/de_DE.json

```diff
@@ -304,23 +304,26 @@
     "tutorial_send_module_009": "Vielen Dank für deine Teilnahme. Wissenschaftler werden dir, soweit möglich, mitteilen, welche Stechmücke du über das Benachrichtigungssystem von Mosquito Alert an sie gesendet hast. Dies kann jedoch je nach Forschungsplanung eine Weile dauern.",
     "text_next_icons": "Du kannst diese Stechmücke per Post verschicken an",
     "ver-laboratorio-de-referencia": "siehe Referenzlabor",
-    "more_info": "Weitere Informationen:",
+    "more_info": "Weitere Informationen",
     "you_can_send_info_address": "Du kannst die Stechmücke an folgende Adresse schicken",
     "you_can_send_report_id": "Verwende diese ID",
-    "identifier_small": "ID:",
+    "identifier_small": "ID",
     "question_17": "Kannst du Larven von Stechmücken sehen?",
     "coordination_txt": "Koordination",
     "partners_txt": "Partner",
-    "disable_auto_hashtag": "Turn off auto-hashtag",
-    "enable_auto_hashtag": "Turn on auto-hashtag",
-    "disable_auto_hashtag_text": "Auto-hashtag will be turned off. Are you sure?",
-    "enable_auto_hashtag_text": "This is an automatic way of adding a specific hashtag to all your records. Use it only if you want to always have your records identified  by a unique hashtag.",
-    "enable_auto_hashtag_hint": "Write your hashtag.\nRemember not to include personal information.",
-    "save": "Save",
-    "ma_not_available": "Looks like Mosquito Alert isn't available right now. Please, try again later.",
-    "not_enough_permissions_location": "We don't have enough permissions to acess the location data on your device. You can enable this on the app settings. Want to go there?",
+    "disable_auto_hashtag": "",
+    "enable_auto_hashtag": "",
+    "disable_auto_hashtag_text": "",
+    "enable_auto_hashtag_text": "",
```

Comment:
The translate() method defaults to english when the string is null, but in this case is empty so it just shows it
https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/blob/de67a3d10ee7afb955923592ed74da1fbaea6cd1/lib/utils/MyLocalizations.dart#L30-L32
![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/09619d61-3f8f-4830-b63c-947ca87b52fc)
![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/ccddb798-9dd3-4fe8-9b90-c64e0f9619fd)

So if we're going to have the empty strings in the json, we have to adapt it
```
String translate(String? key) { 
    if (_localisedValues[key] != null && _localisedValues[key].isNotEmpty) { return _localisedValues[key] }
   return _englishValues[key] ?? ''; 
} 
```

Reply:
Fixed in [d5c7229](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/86/commits/d5c722996314fdc70adf5ac97c02290e9332a9f2)